Exiled.Events/Patches/Events/Player/Joined.cs

```diff
@@ -47,6 +47,7 @@ internal static void CallEvent(ReferenceHub hub, out Player player)
                 }
                 else
                 {
+                    Player.UnverifiedPlayers.Add(hub, player);
```

Comment:
Will bork things.

Reply:
how so?

Reply:
cuz UnverifiedPlayers is a Dictionary<GameObject, Player> not <ReferenceHub, Player> now, and I don't see anywhere that's reverted that change.